extensions/ql-vscode/src/model-editor/modeling-store.ts

```diff
@@ -112,6 +112,10 @@ export class ModelingStore extends DisposableObject {
     return this.state.size > 0;
   }
 
+  public isDbOpen(dbUri: string): boolean {
+    return this.state.has(dbUri);
```

Comment:
What is the difference between the dbUri and the activeDb? Does 'isDbOpen' mean: there's a model editor open for this db?

Reply:
Yeah that's correct. The modeling store knows about dbs rather than model editors so it made sense to keep the terminology consistent there.

Reply:
I think it's a bit weird but open to getting used to it :D